micro_web_server/server.rb

```diff
@@ -0,0 +1,59 @@
+require 'sinatra'
+require_relative 'wishlist_item'
+require 'pg'
+
+DB = PG.connect(
+  dbname: 'sinatra_web_page',
+  user: 'martinjavierwildbaum',
+  password: 'PedroBerro819'
+)
+
+set :public_folder, File.dirname(__FILE__) + '/public'
+set :views, File.join(File.dirname(__FILE__), 'views')
+
+$items = WishlistItem.all
+$budget = 1000
+
+def sort_items_by_cost
+  $items.sort_by!(&:cost)
+end
+
+def check_affordable_items
+  available_budget = $budget
+  $items.each do |item|
+    if item.cost <= available_budget
+      item.completed = true
+      available_budget -= item.cost
+    else
+      item.completed = false
+    end
+  end
+end
+
+get '/' do
+  erb :index
+end
+
+post '/add_item' do
+  description = params[:description]
+  cost = params[:cost].to_f
+
+  if !description.empty? && cost >= 0
```

Comment:
you can use description.present? instead

Reply:
I believe that in order to use the method .present? I need an extra gem, while with .empty I do not

Reply:
What about any?